transaction/_transaction.py

```diff
@@ -138,6 +138,8 @@ def user(self):
     def user(self, v):
         if v is not None:
             self._user = text_or_warn(v)
+        else:
```

Comment:
Assigning None and getting an empty string seems like a bug. :)

This is coming from someone saying that Plone sometimes assigns ``None``.  See the "[dev] RFC: Transaction meta data" thread on the ZODB list where I just posted.

https://groups.google.com/d/msg/zodb/U0H1kTh_dY4/jXg9VFGEBQAJ

Let's discuss the right thing to do there.